src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+---
+title: Create and edit alert conditions in the UI
```

Comment:
this title could capture the essence of your doc a bit better. To me, your doc is about taking a basic alert and adding alert conditions to modify the frequency (or whatever) of that basic alert. The title almost implies that 'in the UI' is the big subject of the doc, but that doesn't come across. Rather, this doc reads like the "step 2" of creating an alert, where the other advanced config doc is like a "step 3". Maybe a title could be like, idk, 'Modify a basic alert with alert conditions' ... You could def think of a better title but something along those lines.